microcloud/cmd/microcloud/shutdown.go

```diff
@@ -14,7 +14,7 @@ type cmdShutdown struct {
 func (c *cmdShutdown) Command() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "shutdown",
-		Short: "Shutdown MicroCluster daemon",
+		Short: "Shut down the MicroCluster daemon",
```

Comment:
We're talking about MicroCluster instead of MicroCloud here - is that correct?
Also in the context of join tokens?

Would it be clearer to talk about "the MicroCloud cluster"?
Or if not, do we need to add something to the docs about what MicroCluster is?

Reply:
Nope, that's a typo :)

`MicroCluster` is the name of the boilerplate/shared logic we use for all the Micro projects, it should never make it to something user visible.

Reply:
OK, thanks, I'll make a follow-up PR.